### DISCUSSION TITLE
How can i download git lfs 3.5.1 version .exe file ?

### DISCUSSION THREAD

**Discussion by PramodRajBharat** [git-lfs/git-lfs#5714] [Q&A] [answered]
" https://git-lfs.com/ "in this site only 3.4.1 version available, but i want to download 3.5.1 version. 

"https://github.com/git-lfs/git-lfs/releases?utm_source=gitlfs_site&utm_medium=releases_link&utm_campaign=gitlfs " here it is showing only zip file to download, but i want .exe file.

Can anyone please help ?

**Comment by bk2204** [comment, marked as answer]
Hey,

You can go to [the v3.5.1 releases page](https://github.com/git-lfs/git-lfs/releases/tag/v3.5.1) and download what you want.  If you're looking for an installer, there's a Windows installer there as an `.exe`, and there's also zip files which contain the binary as a standalone executable.

**Reply by PramodRajBharat** [reply]
Thanks for the info. Its worked
